homeassistant/components/ps4/strings.json

```diff
@@ -7,7 +7,7 @@
       "mode": {
         "data": {
           "mode": "Config Mode",
-          "ip_address": "IP address (Leave empty if using Auto Discovery)."
+          "ip_address": "[%key:common::config_flow::data::ip%]"
```

Comment:
This removed the extra notice that was there. Would it be confusing for users now this is missing?

Reply:
Added back as `data_description`